features/old/accounts/personal_details.feature

```diff
@@ -35,12 +35,14 @@ Feature: Personal Details
     And I press "Update Details"
     Then I should be on the provider users page
 
-  @wip
   Scenario: Edit personal details with invalid data
+    And a buyer "randomdude" signed up to provider "foo.3scale.localhost"
+    When I log in as "randomdude" on foo.3scale.localhost
```

Comment:
Are you sure this intends to test the developer portal personal details screen? Above there's a feature that tests the provider personal details screen with valid data. This could be intended to tests the same screen but with invalid data.

Reply:
yes

Reply:
I agree with @jlledom 
The rest of the feature tests Personal Details in the admin portal, not developer portal (`/p/admin/user/personal_details`)

An interesting thing though (and I guess that's why the scenario was failing) - these steps (filling in email with an emtpy string) actually results in updating the user successfully!!! (thus removing the email)

While I am not sure it's the desired behavior (I think in general we don't allow users without emails), probably the validation is allowing blanks is for some historical reasons... 

In any case, we can rewrite the test by trying to update the username (which DOES involve a validation), e.g. 

![Screenshot from 2023-07-07 17-09-43](https://github.com/3scale/porta/assets/1270328/cb8371be-c17a-46d6-9760-bc6949abd2b0)

Reply:
Okay I think I interpreted it wrong let me check it again and fix it